# This file is distributed under the same license as the Django package.
#
# Translators:
# Claude Paroz <claude@2xlibre.net>, 2013-2018
# Claude Paroz <claude@2xlibre.net>, 2013
# Jannis Leidel <jannis@leidel.info>, 2011
# mlorant <maxime.lorant@gmail.com>, 2014
msgid ""
msgstr ""
"Project-Id-Version: django\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2019-12-20 20:59+0100\n"
"PO-Revision-Date: 2018-02-27 13:36+0000\n"
"Last-Translator: Claude Paroz <claude@2xlibre.net>\n"
"Language-Team: French (http://www.transifex.com/django/django/language/fr/)\n"
"Language: fr\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n > 1);\n"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\admin.py:46
msgid "Personal info"
msgstr "Informations personnelles"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\admin.py:47
msgid "Permissions"
msgstr "Permissions"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\admin.py:50
msgid "Important dates"
msgstr "Dates importantes"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\admin.py:134
#, python-format
msgid "%(name)s object with primary key %(key)r does not exist."
msgstr "L'objet %(name)s avec la clef primaire %(key)r n'existe pas."

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\admin.py:144
msgid "Password changed successfully."
msgstr "Mot de passe modifié avec succès"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\admin.py:164
#, python-format
msgid "Change password: %s"
msgstr "Modifier le mot de passe : %s"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\apps.py:15
msgid "Authentication and Authorization"
msgstr "Authentification et autorisation"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\base_user.py:48
msgid "password"
msgstr "mot de passe"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\base_user.py:49
msgid "last login"
msgstr "dernière connexion"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\forms.py:31
msgid "No password set."
msgstr "Aucun mot de passe défini."

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\forms.py:36
msgid "Invalid password format or unknown hashing algorithm."
msgstr ""
"Format de mot de passe non valide ou algorithme de hachage non reconnu."

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\forms.py:71
#: .\piw\lib\python3.6\site-packages\django\contrib\auth\forms.py:305
#: .\piw\lib\python3.6\site-packages\django\contrib\auth\forms.py:378
msgid "The two password fields didn't match."
msgstr "Les deux mots de passe ne correspondent pas."

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\forms.py:74
#: .\piw\lib\python3.6\site-packages\django\contrib\auth\forms.py:127
#: .\piw\lib\python3.6\site-packages\django\contrib\auth\forms.py:163
#: .\piw\lib\python3.6\site-packages\django\contrib\auth\forms.py:382
msgid "Password"
msgstr "Mot de passe"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\forms.py:80
msgid "Password confirmation"
msgstr "Confirmation du mot de passe"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\forms.py:83
#: .\piw\lib\python3.6\site-packages\django\contrib\auth\forms.py:391
msgid "Enter the same password as before, for verification."
msgstr "Saisissez le même mot de passe que précédemment, pour vérification."

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\forms.py:129
msgid ""
"Raw passwords are not stored, so there is no way to see this user's "
"password, but you can change the password using <a href=\"{}\">this form</a>."
msgstr ""
"Les mots de passe ne sont pas enregistrés en clair, ce qui ne permet pas "
"d'afficher le mot de passe de cet utilisateur, mais il est possible de le "
"changer en utilisant <a href=\"{}\">ce formulaire</a>. "

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\forms.py:170
#, python-format
msgid ""
"Please enter a correct %(username)s and password. Note that both fields may "
"be case-sensitive."
msgstr ""
"Saisissez un %(username)s et un mot de passe valides. Remarquez que chacun "
"de ces champs est sensible à la casse (différenciation des majuscules/"
"minuscules)."

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\forms.py:173
msgid "This account is inactive."
msgstr "Ce compte est inactif."

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\forms.py:233
msgid "Email"
msgstr "Courriel"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\forms.py:308
msgid "New password"
msgstr "Nouveau mot de passe"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\forms.py:314
msgid "New password confirmation"
msgstr "Confirmation du nouveau mot de passe"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\forms.py:350
msgid "Your old password was entered incorrectly. Please enter it again."
msgstr "Votre ancien mot de passe est incorrect. Veuillez le rectifier."

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\forms.py:353
msgid "Old password"
msgstr "Ancien mot de passe"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\forms.py:388
msgid "Password (again)"
msgstr "Mot de passe (à nouveau)"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\hashers.py:260
#: .\piw\lib\python3.6\site-packages\django\contrib\auth\hashers.py:334
#: .\piw\lib\python3.6\site-packages\django\contrib\auth\hashers.py:430
#: .\piw\lib\python3.6\site-packages\django\contrib\auth\hashers.py:490
#: .\piw\lib\python3.6\site-packages\django\contrib\auth\hashers.py:521
#: .\piw\lib\python3.6\site-packages\django\contrib\auth\hashers.py:557
#: .\piw\lib\python3.6\site-packages\django\contrib\auth\hashers.py:593
#: .\piw\lib\python3.6\site-packages\django\contrib\auth\hashers.py:631
msgid "algorithm"
msgstr "algorithme"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\hashers.py:261
msgid "iterations"
msgstr "itérations"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\hashers.py:262
#: .\piw\lib\python3.6\site-packages\django\contrib\auth\hashers.py:340
#: .\piw\lib\python3.6\site-packages\django\contrib\auth\hashers.py:432
#: .\piw\lib\python3.6\site-packages\django\contrib\auth\hashers.py:491
#: .\piw\lib\python3.6\site-packages\django\contrib\auth\hashers.py:522
#: .\piw\lib\python3.6\site-packages\django\contrib\auth\hashers.py:632
msgid "salt"
msgstr "salage"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\hashers.py:263
#: .\piw\lib\python3.6\site-packages\django\contrib\auth\hashers.py:341
#: .\piw\lib\python3.6\site-packages\django\contrib\auth\hashers.py:492
#: .\piw\lib\python3.6\site-packages\django\contrib\auth\hashers.py:523
#: .\piw\lib\python3.6\site-packages\django\contrib\auth\hashers.py:558
#: .\piw\lib\python3.6\site-packages\django\contrib\auth\hashers.py:594
#: .\piw\lib\python3.6\site-packages\django\contrib\auth\hashers.py:633
msgid "hash"
msgstr "empreinte"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\hashers.py:335
msgid "variety"
msgstr "variété"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\hashers.py:336
msgid "version"
msgstr "version"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\hashers.py:337
msgid "memory cost"
msgstr "coût mémoire"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\hashers.py:338
msgid "time cost"
msgstr "coût temps"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\hashers.py:339
msgid "parallelism"
msgstr "parallélisme"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\hashers.py:431
msgid "work factor"
msgstr "facteur travail"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\hashers.py:433
msgid "checksum"
msgstr "somme de contrôle"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\models.py:56
#: .\piw\lib\python3.6\site-packages\django\contrib\auth\models.py:112
msgid "name"
msgstr "nom"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\models.py:60
msgid "content type"
msgstr "type de contenu"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\models.py:62
msgid "codename"
msgstr "nom de code"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\models.py:67
msgid "permission"
msgstr "permission"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\models.py:68
#: .\piw\lib\python3.6\site-packages\django\contrib\auth\models.py:115
msgid "permissions"
msgstr "permissions"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\models.py:122
msgid "group"
msgstr "groupe"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\models.py:123
#: .\piw\lib\python3.6\site-packages\django\contrib\auth\models.py:219
msgid "groups"
msgstr "groupes"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\models.py:210
msgid "superuser status"
msgstr "statut super-utilisateur"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\models.py:213
msgid ""
"Designates that this user has all permissions without explicitly assigning "
"them."
msgstr ""
"Précise que l'utilisateur possède toutes les permissions sans les assigner "
"explicitement."

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\models.py:222
msgid ""
"The groups this user belongs to. A user will get all permissions granted to "
"each of their groups."
msgstr ""
"Les groupes dont fait partie cet utilisateur. Celui-ci obtient tous les "
"droits de tous les groupes auxquels il appartient."

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\models.py:230
msgid "user permissions"
msgstr "permissions de l'utilisateur"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\models.py:232
msgid "Specific permissions for this user."
msgstr "Permissions spécifiques à cet utilisateur."

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\models.py:299
msgid "username"
msgstr "nom d'utilisateur"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\models.py:302
msgid "Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only."
msgstr ""
"Requis. 150 caractères maximum. Uniquement des lettres, nombres et les "
"caractères « @ », « . », « + », « - » et « _ »."

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\models.py:305
msgid "A user with that username already exists."
msgstr "Un utilisateur avec ce nom existe déjà."

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\models.py:308
msgid "first name"
msgstr "prénom"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\models.py:309
msgid "last name"
msgstr "nom"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\models.py:310
msgid "email address"
msgstr "adresse électronique"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\models.py:312
msgid "staff status"
msgstr "statut équipe"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\models.py:314
msgid "Designates whether the user can log into this admin site."
msgstr "Précise si l'utilisateur peut se connecter à ce site d'administration."

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\models.py:317
msgid "active"
msgstr "actif"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\models.py:320
msgid ""
"Designates whether this user should be treated as active. Unselect this "
"instead of deleting accounts."
msgstr ""
"Précise si l'utilisateur doit être considéré comme actif. Décochez ceci "
"plutôt que de supprimer le compte."

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\models.py:324
msgid "date joined"
msgstr "date d'inscription"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\models.py:333
msgid "user"
msgstr "utilisateur"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\models.py:334
msgid "users"
msgstr "utilisateurs"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\password_validation.py:102
#, python-format
msgid ""
"This password is too short. It must contain at least %(min_length)d "
"character."
msgid_plural ""
"This password is too short. It must contain at least %(min_length)d "
"characters."
msgstr[0] ""
"Ce mot de passe est trop court. Il doit contenir au minimum %(min_length)d "
"caractère."
msgstr[1] ""
"Ce mot de passe est trop court. Il doit contenir au minimum %(min_length)d "
"caractères."

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\password_validation.py:112
#, python-format
msgid "Your password must contain at least %(min_length)d character."
msgid_plural "Your password must contain at least %(min_length)d characters."
msgstr[0] ""
"Votre mot de passe doit contenir au minimum %(min_length)d caractère."
msgstr[1] ""
"Votre mot de passe doit contenir au minimum %(min_length)d caractères."

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\password_validation.py:151
#, python-format
msgid "The password is too similar to the %(verbose_name)s."
msgstr "Le mot de passe est trop semblable au champ « %(verbose_name)s »."

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\password_validation.py:157
msgid "Your password can't be too similar to your other personal information."
msgstr ""
"Votre mot de passe ne peut pas trop ressembler à vos autres informations "
"personnelles."

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\password_validation.py:185
msgid "This password is too common."
msgstr "Ce mot de passe est trop courant."

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\password_validation.py:190
msgid "Your password can't be a commonly used password."
msgstr ""
"Votre mot de passe ne peut pas être un mot de passe couramment utilisé."

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\password_validation.py:200
msgid "This password is entirely numeric."
msgstr "Ce mot de passe est entièrement numérique."

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\password_validation.py:205
msgid "Your password can't be entirely numeric."
msgstr "Votre mot de passe ne peut pas être entièrement numérique."

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\templates\registration\password_reset_subject.txt:2
#, python-format
msgid "Password reset on %(site_name)s"
msgstr "Réinitialisation du mot de passe sur %(site_name)s"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\validators.py:12
msgid ""
"Enter a valid username. This value may contain only English letters, "
"numbers, and @/./+/-/_ characters."
msgstr ""
"Saisissez un nom d'utilisateur valide. Il ne peut contenir que des lettres "
"non accentuées, des nombres ou les caractères « @ », « . », « + », « - » et "
"« _ »."

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\validators.py:22
msgid ""
"Enter a valid username. This value may contain only letters, numbers, and "
"@/./+/-/_ characters."
msgstr ""
"Saisissez un nom d'utilisateur valide. Il ne peut contenir que des lettres, "
"des nombres ou les caractères « @ », « . », « + », « - » et « _ »."

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\views.py:158
msgid "Logged out"
msgstr "Déconnecté"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\views.py:215
msgid "Password reset"
msgstr "Réinitialisation du mot de passe"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\views.py:243
msgid "Password reset sent"
msgstr "Message de réinitialisation du mot de passe envoyé"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\views.py:252
msgid "Enter new password"
msgstr "Saisissez un nouveau mot de passe"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\views.py:312
msgid "Password reset unsuccessful"
msgstr "Échec lors de la mise à jour du mot de passe"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\views.py:320
msgid "Password reset complete"
msgstr "Mise à jour du mot de passe terminée"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\views.py:332
msgid "Password change"
msgstr "Modification du mot de passe"

#: .\piw\lib\python3.6\site-packages\django\contrib\auth\views.py:355
msgid "Password change successful"
msgstr "Mot de passe modifié avec succès"
